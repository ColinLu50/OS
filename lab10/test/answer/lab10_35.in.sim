33306
905
923
635
905
624
