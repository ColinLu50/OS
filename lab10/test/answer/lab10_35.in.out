FCFS
273 -> 423 -> 259 -> 430 -> 603 -> 480 -> 9 -> 85 -> 359 -> 106 -> 94 -> 137 -> 62 -> 519 -> 444 -> 392 -> 648 -> 650 -> 71 -> 593 -> 198 -> 565 -> 625 -> 48 -> 361 -> 85 -> 168 -> 496 -> 354 -> 625 -> 346 -> 613 -> 415 -> 358 -> 215 -> 345 -> 88 -> 81 -> 392 -> 220 -> 574 -> 126 -> 86 -> 608 -> 249 -> 535 -> 306 -> 425 -> 15 -> 351 -> 218 -> 357 -> 320 -> 617 -> 444 -> 482 -> 297 -> 417 -> 597 -> 258 -> 358 -> 574 -> 149 -> 239 -> 12 -> 449 -> 161 -> 333 -> 105 -> 309 -> 566 -> 535 -> 68 -> 66 -> 435 -> 197 -> 202 -> 578 -> 530 -> 312 -> 150 -> 117 -> 244 -> 27 -> 407 -> 117 -> 119 -> 60 -> 450 -> 342 -> 190 -> 329 -> 238 -> 290 -> 585 -> 322 -> 340 -> 52 -> 442 -> 237 -> 80 -> 434 -> 52 -> 526 -> 28 -> 612 -> 113 -> 379 -> 30 -> 343 -> 578 -> 471 -> 142 -> 395 -> 551 -> 547 -> 516 -> 490 -> 311 -> 375 -> 577 -> 512 -> 184 -> 238 -> 208 -> 409 -> 52 -> 606 -> 310 -> 371 -> 461 -> 538 -> 126 -> 604 -> 236 -> 68 -> 414 -> 619 -> 447 -> 86 -> 505 -> 111 -> 404 -> 41 -> 253 -> 77
33306
SSTF
273 -> 259 -> 258 -> 253 -> 249 -> 244 -> 239 -> 238 -> 238 -> 237 -> 236 -> 220 -> 218 -> 215 -> 208 -> 202 -> 198 -> 197 -> 190 -> 184 -> 168 -> 161 -> 150 -> 149 -> 142 -> 137 -> 126 -> 126 -> 119 -> 117 -> 117 -> 113 -> 111 -> 106 -> 105 -> 94 -> 88 -> 86 -> 86 -> 85 -> 85 -> 81 -> 80 -> 77 -> 71 -> 68 -> 68 -> 66 -> 62 -> 60 -> 52 -> 52 -> 52 -> 48 -> 41 -> 30 -> 28 -> 27 -> 15 -> 12 -> 9 -> 290 -> 297 -> 306 -> 309 -> 310 -> 311 -> 312 -> 320 -> 322 -> 329 -> 333 -> 340 -> 342 -> 343 -> 345 -> 346 -> 351 -> 354 -> 357 -> 358 -> 358 -> 359 -> 361 -> 371 -> 375 -> 379 -> 392 -> 392 -> 395 -> 404 -> 407 -> 409 -> 414 -> 415 -> 417 -> 423 -> 425 -> 430 -> 434 -> 435 -> 442 -> 444 -> 444 -> 447 -> 449 -> 450 -> 461 -> 471 -> 480 -> 482 -> 490 -> 496 -> 505 -> 512 -> 516 -> 519 -> 526 -> 530 -> 535 -> 535 -> 538 -> 547 -> 551 -> 565 -> 566 -> 574 -> 574 -> 577 -> 578 -> 578 -> 585 -> 593 -> 597 -> 603 -> 604 -> 606 -> 608 -> 612 -> 613 -> 617 -> 619 -> 625 -> 625 -> 648 -> 650
905
SCAN
273 -> 259 -> 258 -> 253 -> 249 -> 244 -> 239 -> 238 -> 237 -> 236 -> 220 -> 218 -> 215 -> 208 -> 202 -> 198 -> 197 -> 190 -> 184 -> 168 -> 161 -> 150 -> 149 -> 142 -> 137 -> 126 -> 119 -> 117 -> 113 -> 111 -> 106 -> 105 -> 94 -> 88 -> 86 -> 85 -> 81 -> 80 -> 77 -> 71 -> 68 -> 66 -> 62 -> 60 -> 52 -> 48 -> 41 -> 30 -> 28 -> 27 -> 15 -> 12 -> 9 -> 0 -> 290 -> 297 -> 306 -> 309 -> 310 -> 311 -> 312 -> 320 -> 322 -> 329 -> 333 -> 340 -> 342 -> 343 -> 345 -> 346 -> 351 -> 354 -> 357 -> 358 -> 359 -> 361 -> 371 -> 375 -> 379 -> 392 -> 395 -> 404 -> 407 -> 409 -> 414 -> 415 -> 417 -> 423 -> 425 -> 430 -> 434 -> 435 -> 442 -> 444 -> 447 -> 449 -> 450 -> 461 -> 471 -> 480 -> 482 -> 490 -> 496 -> 505 -> 512 -> 516 -> 519 -> 526 -> 530 -> 535 -> 538 -> 547 -> 551 -> 565 -> 566 -> 574 -> 577 -> 578 -> 585 -> 593 -> 597 -> 603 -> 604 -> 606 -> 608 -> 612 -> 613 -> 617 -> 619 -> 625 -> 648 -> 650
923
C-SCAN
273 -> 259 -> 258 -> 253 -> 249 -> 244 -> 239 -> 238 -> 237 -> 236 -> 220 -> 218 -> 215 -> 208 -> 202 -> 198 -> 197 -> 190 -> 184 -> 168 -> 161 -> 150 -> 149 -> 142 -> 137 -> 126 -> 119 -> 117 -> 113 -> 111 -> 106 -> 105 -> 94 -> 88 -> 86 -> 85 -> 81 -> 80 -> 77 -> 71 -> 68 -> 66 -> 62 -> 60 -> 52 -> 48 -> 41 -> 30 -> 28 -> 27 -> 15 -> 12 -> 9 -> 0 -> 652 -> 650 -> 648 -> 625 -> 619 -> 617 -> 613 -> 612 -> 608 -> 606 -> 604 -> 603 -> 597 -> 593 -> 585 -> 578 -> 577 -> 574 -> 566 -> 565 -> 551 -> 547 -> 538 -> 535 -> 530 -> 526 -> 519 -> 516 -> 512 -> 505 -> 496 -> 490 -> 482 -> 480 -> 471 -> 461 -> 450 -> 449 -> 447 -> 444 -> 442 -> 435 -> 434 -> 430 -> 425 -> 423 -> 417 -> 415 -> 414 -> 409 -> 407 -> 404 -> 395 -> 392 -> 379 -> 375 -> 371 -> 361 -> 359 -> 358 -> 357 -> 354 -> 351 -> 346 -> 345 -> 343 -> 342 -> 340 -> 333 -> 329 -> 322 -> 320 -> 312 -> 311 -> 310 -> 309 -> 306 -> 297 -> 290
635
LOOK
273 -> 259 -> 258 -> 253 -> 249 -> 244 -> 239 -> 238 -> 237 -> 236 -> 220 -> 218 -> 215 -> 208 -> 202 -> 198 -> 197 -> 190 -> 184 -> 168 -> 161 -> 150 -> 149 -> 142 -> 137 -> 126 -> 119 -> 117 -> 113 -> 111 -> 106 -> 105 -> 94 -> 88 -> 86 -> 85 -> 81 -> 80 -> 77 -> 71 -> 68 -> 66 -> 62 -> 60 -> 52 -> 48 -> 41 -> 30 -> 28 -> 27 -> 15 -> 12 -> 9 -> 290 -> 297 -> 306 -> 309 -> 310 -> 311 -> 312 -> 320 -> 322 -> 329 -> 333 -> 340 -> 342 -> 343 -> 345 -> 346 -> 351 -> 354 -> 357 -> 358 -> 359 -> 361 -> 371 -> 375 -> 379 -> 392 -> 395 -> 404 -> 407 -> 409 -> 414 -> 415 -> 417 -> 423 -> 425 -> 430 -> 434 -> 435 -> 442 -> 444 -> 447 -> 449 -> 450 -> 461 -> 471 -> 480 -> 482 -> 490 -> 496 -> 505 -> 512 -> 516 -> 519 -> 526 -> 530 -> 535 -> 538 -> 547 -> 551 -> 565 -> 566 -> 574 -> 577 -> 578 -> 585 -> 593 -> 597 -> 603 -> 604 -> 606 -> 608 -> 612 -> 613 -> 617 -> 619 -> 625 -> 648 -> 650
905
C-LOOK
273 -> 259 -> 258 -> 253 -> 249 -> 244 -> 239 -> 238 -> 237 -> 236 -> 220 -> 218 -> 215 -> 208 -> 202 -> 198 -> 197 -> 190 -> 184 -> 168 -> 161 -> 150 -> 149 -> 142 -> 137 -> 126 -> 119 -> 117 -> 113 -> 111 -> 106 -> 105 -> 94 -> 88 -> 86 -> 85 -> 81 -> 80 -> 77 -> 71 -> 68 -> 66 -> 62 -> 60 -> 52 -> 48 -> 41 -> 30 -> 28 -> 27 -> 15 -> 12 -> 9 -> 650 -> 648 -> 625 -> 619 -> 617 -> 613 -> 612 -> 608 -> 606 -> 604 -> 603 -> 597 -> 593 -> 585 -> 578 -> 577 -> 574 -> 566 -> 565 -> 551 -> 547 -> 538 -> 535 -> 530 -> 526 -> 519 -> 516 -> 512 -> 505 -> 496 -> 490 -> 482 -> 480 -> 471 -> 461 -> 450 -> 449 -> 447 -> 444 -> 442 -> 435 -> 434 -> 430 -> 425 -> 423 -> 417 -> 415 -> 414 -> 409 -> 407 -> 404 -> 395 -> 392 -> 379 -> 375 -> 371 -> 361 -> 359 -> 358 -> 357 -> 354 -> 351 -> 346 -> 345 -> 343 -> 342 -> 340 -> 333 -> 329 -> 322 -> 320 -> 312 -> 311 -> 310 -> 309 -> 306 -> 297 -> 290
624
